helm/idol-omnigroupserver/values.yaml

```diff
@@ -61,6 +75,17 @@ servicePort: "3058"
 livenessProbe:
   initialDelaySeconds: 30
 
+additionalVolumes:
+- name: oauth-startup-scripts
+  configMap:
+    name: omnigroupserver-oauth-startup-scripts-cm
+
+additionalVolumeMounts:
+- name: oauth-startup-scripts
+  mountPath: /omnigroupserver/prestart_scripts
+  readOnly: true
```

Comment:
These I guess we can remove once we add the idol-library deployment hooks

Reply:
[eaee33f](https://github.com/opentext-idol/idol-containers-toolkit/pull/31/commits/eaee33f41850e811f1221d6b2d6d9f345bf1821b)